var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -8,10 +8,39 @@
 
 class Visit(CMakePackage):
     """VisIt is an Open Source, interactive, scalable, visualization,
-       animation and analysis tool."""
+       animation and analysis tool.
+
+    ############################
+    # Suggestions for building.
+    ############################
+    # cyrush note:
+    #
+    # Out of the box, VisIt's python 2 requirement will cause
+    # spack spec constraint errors due Qt + Mesa build
+    # dependencies.
+    #
+    # You can avoid this using:
+    #
+    # linux:
+    #  spack install visit ^python+shared ^glib@2.56.3 ^py-setuptools@44.1.0
+    #
+    # macOS:
+    #  spack install visit ^python+shared ^glib@2.56.3 ^py-setuptools@44.1.0 \
+    #                      ^qt~framework
+    #
+    # Rpath issues undermine qwt (not qt) when a build as a framework
+    # VisIt's osxfixup resolves this for us in other cases,
+    # but we can't use osxfixup with spack b/c it will undermine other libs.
+    #
+    # Even with these changes, VisIt's Python CLI does not work on macOS,
+    # there is a linking issue related to OpenSSL.
+    # (dyld: Symbol not found: _GENERAL_NAME_free - which comes from OpenSSL)
+    #
+    """
```

Comment:
`spack info visit` doesn't handle formatting well... All of this will end up being concatenated. It might be better to put this in as a comment instead of as a docstring.

Reply:
yes, I see -- thanks I changed the doc string and moved to comments